incubator/hnc/internal/validators/object_test.go

```diff
@@ -146,6 +146,7 @@ func TestUserChanges(t *testing.T) {
 	f := forest.NewForest()
 	o := &Object{Forest: f}
 	l := zap.Logger(false)
+	invalidSelStr := "1234567890123456789012345678901234567890123456789012345678901234567890"
```

Comment:
This only gets used once so I'd recommend inlining it into the test.

Reply:
I'm thinking of using it in the selector test as well

Reply:
Ah, I see. I think it might be better if each test had its own, different, bad selector :) E.g. "a b c" is a bad namespace name _and_ a bad selector.

Constants like this make the tests harder to read, because you can't just look at the test definition to know what's being tested. Constants can add clarity in same cases (e.g. you don't really have to look inside RBACGroup to know what's there) but "invalidSelector" could have anything in there, so it makes the test a bit harder to read IMO.

I know I sometimes say "please use constants!" and it's weird that I'm suggesting _not_ using them here, but I hope that explains why!

Reply:
I see what you mean. Yeah I agree, changed the test string here